conf/task_defaults/chesapeake_cvpr_prior.yaml

```diff
@@ -0,0 +1,29 @@
+experiment:
+  task: "chesapeake_cvpr"
+  module:
+    loss: "ce"
+    segmentation_model: "unet"
+    encoder_name: "resnet50"
+    encoder_weights: null
+    encoder_output_stride: 16
+    learning_rate: 1e-3
+    learning_rate_schedule_patience: 6
+    in_channels: 4
+    num_classes: 5
+    num_filters: 1
+    ignore_zeros: False
+    imagenet_pretraining: False
+  datamodule:
+    root_dir: "tests/data/chesapeake/cvpr"
+    train_splits:
+    - "de-test"
+    val_splits:
+    - "de-test"
+    test_splits:
+    - "de-test"
+    patches_per_tile: 2
+    patch_size: 64
+    batch_size: 2
+    num_workers: 0
+    class_set: ${experiment.module.num_classes}
+    use_prior_labels: True
```

Comment:
These datamodule settings (`use_prior_labels: True`) work with BYOLTask but not with SemanticSegmentationTask and I have no idea why.

Reply:
Not sure what error you're getting but BYOLTask doesn't use the masks so not surprised it passes. The prior labels I believe are soft probabilities so I don't think we've set up the SegmentationTask loss to handle that.